source/nodejs/adaptivecards-designer-app/src/app.ts

```diff
@@ -12,13 +12,11 @@ import "./app.css";
 
 window.onload = function() {
     // Uncomment to enabled preview features such as data binding
-    /*
```

Comment:
Are we uncommenting this to expose the databinding feature by default now?

Reply:
Yes. Also keep in mind that adaptivecards-designer-app is basically a test app.